backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -404,6 +406,51 @@ public RequestStatus updateProject(Project project, User user) throws SW360Excep
         }
     }
 
+    private void setRequestedDateAndTrimComment(Project project, Project actual, User user) {
+        Set<String> actualReleaseIds = null;
+        if (Objects.nonNull(actual)&&Objects.nonNull(actual.getReleaseIdToUsage())) {
+            actualReleaseIds = CommonUtils.nullToEmptySet(actual.getReleaseIdToUsage().keySet());
+        }
+        final Set<String> actualReleaseIdsFinal = CommonUtils.nullToEmptySet(actualReleaseIds);
+        Set<String> updatedReleaseIds = null;
+        if (Objects.nonNull(project.getReleaseIdToUsage())) {
+            updatedReleaseIds = CommonUtils.nullToEmptySet(project.getReleaseIdToUsage().keySet());
+        } else {
+            updatedReleaseIds = new HashSet<>();
+        }
+
+        updatedReleaseIds.stream().filter(updatedReleaseId -> !actualReleaseIdsFinal.contains(updatedReleaseId))
+                .forEach(updatedReleaseId -> {
+                    ProjectReleaseRelationship projectReleaseRelationship = project.getReleaseIdToUsage()
+                            .get(updatedReleaseId);
+                    if (Objects.nonNull(projectReleaseRelationship))
+                        projectReleaseRelationship.setCreatedOn(SW360Utils.getCreatedOn());
+                    projectReleaseRelationship.setCreatedBy(user.getEmail());
+                });
+
+        updatedReleaseIds.stream().filter(commonReleaseId -> actualReleaseIdsFinal.contains(commonReleaseId))
+                .forEach(commonReleaseId -> {
```

Comment:
This is a lot of effort to keep track of `createdOn` and `createdBy` field for `linked releases`.
I would suggest to keep these fields (`createdOn` and `createdBy`) as hidden form fields, and un-hide it before form submission. So that we don't need this logic in back-end, as it would be cumbersome for projects with 100's of `linked releases`.